download/download_test.go

```diff
@@ -0,0 +1,40 @@
+package download
+
+import (
+	"testing"
+
+	"github.com/alexandrasp/taskcluster-cli/extpoints"
```

Comment:
you should add `github.com/alexandrasp/taskcluster-cli` as a remote `git remote add alexandrasp github.com/alexandrasp/taskcluster-cli` to the clone you have in `github.com/taskcluster/taskcluster-cli` to avoid issues like this...